src/devices/ovrheadset/OVRHeadset.cpp

```diff
@@ -33,7 +33,7 @@
 #include <dxgi.h> // for GetDefaultAdapterLuid
 #pragma comment(lib, "dxgi.lib")
 #endif
-
+#define M_PI 3.141
```

Comment:
If you want to use the non-standard `M_PI` constant, please add the `_USE_MATH_DEFINES` define in Windows, see https://github.com/robotology/QA/issues/81 .